composer.json

```diff
@@ -2,7 +2,10 @@
     "name": "prismic/php-sdk",
     "description": "PHP development kit for Prismic",
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "allow-plugins": {
+            "symfony/flex": true
+        }
```

Comment:
When does the symfony flex plugin run? I don't see it in deps

Reply:
That must have somehow snuck its way into this. Removed in this commit [5b13305](https://github.com/prismicio-community/php-kit/pull/190/commits/5b13305a41338d1815b05ddf59521882e9fa0246)